test/e2e_node/services/kubelet.go

```diff
@@ -63,6 +63,23 @@ func init() {
 	flag.Var(&kubeletArgs, "kubelet-flags", "Kubelet flags passed to kubelet, this will override default kubelet flags in the test. Flags specified in multiple kubelet-flags will be concatenate.")
 }
 
+// RunKubelet starts kubelet and waits for termination signal. Once receives the
+// termination signal, it will stop the kubelet gracefully.
+func RunKubelet() error {
```

Comment:
nit: return unused. Either remove it, or return the error and call glog.Fatal in the caller.